src/components/main/footer.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import Link from "next/link";
+
+const footerItemsA = [
+    { name: 'Mission Statement', href: '/statement/' },
+    { name: 'Branding', href: 'https://wiki.rhinolinux.org/dev/brand' },
+];
+
+const footerItemsB = [
+    { name: 'Code of Conduct', href: '/code/'},
+    { name: 'Donate', href: 'https://ko-fi.com/rhinolinux' },
+];
+
+function Footer() {
+    return(
+        <footer className="p-8">
+            <hr className="md:w-[85%] h-1 mx-auto my-4 bg-site-300 border-0 rounded" />
+            <div className="md:w-[85%] m-auto flex items-center justify-center">
```

Comment:
All footer items should stack and be center-aligned on mobile

Reply:
That’s way too much space for the footer to take up